docs/setup_installation/gcp/cluster_creation.md

```diff
@@ -25,7 +25,14 @@ Name your cluster (2). Choose the *Region*(3) and *Zone*(4) in which to deploy t
 
 Select the *Instance type* (5) and *Local storage* (6) size for the cluster *Head node*.
 
-Enter the name of the bucket in which the hopsworks cluster will store its data in *Cloud Storage Bucket* (7)
+Optional: Specify a [customer-managed encryption key](https://cloud.google.com/compute/docs/disks/customer-managed-encryption) to be used for encryption of local storage. The key has to be specified using the format: `projects/PROJECT_ID/locations/REGION/keyRings/KEY_RING/cryptoKeys/KEY`. Note that your project needs to be configured to allow usage of the key. This can be achieved by executing the gcloud command below. Refer to the GCP documentation for more details: [Protect resources by using Cloud KMS keys](https://cloud.google.com/compute/docs/disks/customer-managed-encryption#before_you_begin).
+    
+    gcloud projects add-iam-policy-binding KMS_PROJECT_ID \
```

Comment:
Don't you need the backticks (```) for this to be rendered as a code block?

Reply:
The docs said indentation is enough and it looked correctly in my preview. https://www.markdownguide.org/basic-syntax/#code-blocks-1